apps/test/unit/lib/levelbuilder/script-editor/UnitEditorTest.jsx

```diff
@@ -104,15 +105,25 @@ describe('UnitEditor', () => {
     );
   };
 
+  function renderDefault(overrideProps = {}) {
+    const combinedProps = {...defaultProps, ...overrideProps};
+    render(
+      <Provider store={store}>
+        <UnitEditor {...combinedProps} />
+      </Provider>
+    );
+  }
+
   describe('Script Editor', () => {
```

Comment:
Because we aren't migrating all the tests in the file, can we make two `describe` statements?
```suggestion
  describe('Script Editor', () => {
  describe('Script Editor - Legacy (Enzyme)', () => {
```

Reply:
Good suggestion